snuba/datasets/outcomes_raw.py

```diff
@@ -39,6 +56,7 @@ def __init__(self):
             order_by="(org_id, project_id, timestamp)",
             partition_by="(toMonday(timestamp))",
             settings={"index_granularity": 16384},
+            migration_function=outcomes_raw_read_migrations
```

Comment:
This dataset represents a view over the table created by the Outcomes dataset. No need to add the migration. The table is the same already create and migrated by outcomes.

Reply:
Thanks for clarifying, I removed that migration